tools/image_processing/imagej2/imagej2_shadows/imagej2_shadows.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_shadows" name="Add shadow effect" version="@WRAPPER_VERSION@.0">
+    <description>to images with ImageJ2</description>
```

Comment:
@bgruening The fact that we've both questioned whether it is necessary is a sign, I think, that it should be removed.  So I'll remove it in my next PR.  ;)